neo/test/coretest/test_spiketrain.py

```diff
@@ -1147,7 +1155,7 @@ def test_merge_multiple(self):
         expected *= time_unit
         sorting = np.argsort(expected)
         expected = expected[sorting]
-        np.testing.assert_array_equal(result.times, expected)
+        np.testing.assert_array_equal(result.times.magnitude, expected.magnitude)
```

Comment:
This change is necessary because as of numpy 2.0 because numpy assert_array_equal actually checks that the quantities are the same which leads to a units issue. If someone else understands this better please post here.